Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -848,35 +854,28 @@ defineSuite([
             expect(stats.numberOfPendingRequests).toEqual(1);
             scene.primitives.remove(tileset);
 
-            return root.contentReadyPromise.then(function(root) {
-                fail('should not resolve');
-            }).otherwise(function(error) {
-                // Expect the root to not have added any children from the external tileset.json
-                expect(root.children.length).toEqual(0);
-                expect(RequestScheduler.getNumberOfAvailableRequests()).toEqual(RequestScheduler.maximumRequests);
-            });
```

Comment:
What code needs to be covered?  This assertion is no longer true since this PR moved the `contentReadyToProcessPromise` and `contentReadyToProcessPromise` properties and instead rely on promises on the tile's content, e.g.,
```javascript
 -            when(tile.contentReadyToProcessPromise).then(addToProcessingQueue(tileset, tile)).otherwise(removeFunction);
 -            when(tile.contentReadyPromise).then(removeFunction).otherwise(removeFunction);
 +            when(tile.content.contentReadyToProcessPromise).then(addToProcessingQueue(tileset, tile)).otherwise(removeFunction);
 +            when(tile.content.readyPromise).then(removeFunction).otherwise(removeFunction);
```